[2025-07-18T11:49:18.270-0500] {processor.py:161} INFO - Started process (PID=809) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:49:18.270-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T11:49:18.271-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:49:18.271-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:49:18.273-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:49:18.273-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T11:49:18.274-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:49:18.273-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T11:49:18.274-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:49:18.288-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T11:49:49.636-0500] {processor.py:161} INFO - Started process (PID=935) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:49:49.636-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T11:49:49.637-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:49:49.637-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:49:49.639-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:49:49.639-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T11:49:49.639-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:49:49.639-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T11:49:49.640-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:49:49.656-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-07-18T11:50:20.474-0500] {processor.py:161} INFO - Started process (PID=1058) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:50:20.475-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T11:50:20.476-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:50:20.476-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:50:20.478-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:50:20.477-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T11:50:20.478-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:50:20.478-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T11:50:20.478-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:50:20.491-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-07-18T11:50:50.644-0500] {processor.py:161} INFO - Started process (PID=1166) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:50:50.644-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T11:50:50.645-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:50:50.645-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:50:50.647-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:50:50.647-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T11:50:50.648-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:50:50.648-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T11:50:50.648-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:50:50.662-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T11:51:20.797-0500] {processor.py:161} INFO - Started process (PID=1275) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:51:20.799-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T11:51:20.800-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:51:20.799-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:51:20.802-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:51:20.801-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T11:51:20.802-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:51:20.802-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T11:51:20.802-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:51:20.815-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T11:51:51.131-0500] {processor.py:161} INFO - Started process (PID=1380) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:51:51.132-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T11:51:51.133-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:51:51.133-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:51:51.137-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:51:51.136-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T11:51:51.137-0500] {logging_mixin.py:188} INFO - [2025-07-18T11:51:51.137-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T11:51:51.137-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T11:51:51.155-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2025-07-18T12:32:15.478-0500] {processor.py:161} INFO - Started process (PID=1920) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:32:15.480-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:32:15.481-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:32:15.480-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:32:15.482-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:32:15.482-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:32:15.483-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:32:15.482-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:32:15.483-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:32:15.500-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2025-07-18T12:32:45.897-0500] {processor.py:161} INFO - Started process (PID=2050) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:32:45.898-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:32:45.899-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:32:45.899-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:32:45.901-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:32:45.900-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:32:45.901-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:32:45.901-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:32:45.901-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:32:45.919-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-07-18T12:33:16.077-0500] {processor.py:161} INFO - Started process (PID=2169) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:33:16.079-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:33:16.080-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:33:16.080-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:33:16.082-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:33:16.082-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:33:16.083-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:33:16.082-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:33:16.083-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:33:16.099-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2025-07-18T12:33:46.345-0500] {processor.py:161} INFO - Started process (PID=2283) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:33:46.345-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:33:46.346-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:33:46.346-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:33:46.349-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:33:46.348-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:33:46.349-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:33:46.349-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:33:46.349-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:33:46.363-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:34:16.672-0500] {processor.py:161} INFO - Started process (PID=2389) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:34:16.674-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:34:16.675-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:34:16.674-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:34:16.677-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:34:16.676-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:34:16.677-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:34:16.677-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:34:16.677-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:34:16.693-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2025-07-18T12:34:47.286-0500] {processor.py:161} INFO - Started process (PID=2497) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:34:47.286-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:34:47.287-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:34:47.287-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:34:47.290-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:34:47.289-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:34:47.290-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:34:47.290-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:34:47.290-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:34:47.304-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:35:17.721-0500] {processor.py:161} INFO - Started process (PID=2607) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:35:17.722-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:35:17.723-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:35:17.723-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:35:17.725-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:35:17.725-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:35:17.726-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:35:17.726-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:35:17.726-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:35:17.740-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2025-07-18T12:35:48.046-0500] {processor.py:161} INFO - Started process (PID=2712) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:35:48.046-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:35:48.047-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:35:48.047-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:35:48.049-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:35:48.049-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:35:48.049-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:35:48.049-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:35:48.049-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:35:48.068-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2025-07-18T12:36:18.592-0500] {processor.py:161} INFO - Started process (PID=2819) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:36:18.593-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:36:18.594-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:36:18.594-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:36:18.596-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:36:18.595-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:36:18.596-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:36:18.596-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:36:18.596-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:36:18.610-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:36:48.917-0500] {processor.py:161} INFO - Started process (PID=2935) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:36:48.917-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:36:48.918-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:36:48.918-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:36:48.920-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:36:48.920-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:36:48.921-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:36:48.920-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:36:48.921-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:36:48.937-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-07-18T12:37:19.299-0500] {processor.py:161} INFO - Started process (PID=3151) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:37:19.300-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:37:19.301-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:37:19.301-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:37:19.303-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:37:19.303-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:37:19.303-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:37:19.303-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:37:19.304-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:37:19.322-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2025-07-18T12:37:49.735-0500] {processor.py:161} INFO - Started process (PID=3259) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:37:49.736-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:37:49.737-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:37:49.737-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:37:49.739-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:37:49.739-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:37:49.739-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:37:49.739-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:37:49.739-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:37:49.755-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-07-18T12:38:19.949-0500] {processor.py:161} INFO - Started process (PID=3366) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:38:19.951-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:38:19.952-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:38:19.952-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:38:19.954-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:38:19.953-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:38:19.954-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:38:19.954-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:38:19.954-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:38:19.969-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-07-18T12:38:50.456-0500] {processor.py:161} INFO - Started process (PID=3474) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:38:50.458-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:38:50.459-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:38:50.458-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:38:50.460-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:38:50.460-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:38:50.461-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:38:50.461-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:38:50.461-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:38:50.475-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:39:20.709-0500] {processor.py:161} INFO - Started process (PID=3581) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:39:20.710-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:39:20.711-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:39:20.711-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:39:20.713-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:39:20.713-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:39:20.714-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:39:20.714-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:39:20.714-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:39:20.729-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-07-18T12:39:50.957-0500] {processor.py:161} INFO - Started process (PID=3686) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:39:50.958-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:39:50.959-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:39:50.959-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:39:50.961-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:39:50.960-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:39:50.961-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:39:50.961-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:39:50.961-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:39:50.974-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-07-18T12:40:22.069-0500] {processor.py:161} INFO - Started process (PID=3806) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:40:22.070-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:40:22.071-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:40:22.071-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:40:22.074-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:40:22.073-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:40:22.074-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:40:22.074-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:40:22.074-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:40:22.087-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:40:31.250-0500] {processor.py:161} INFO - Started process (PID=3879) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:40:31.250-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:40:31.251-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:40:31.251-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:40:31.253-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:40:31.253-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:40:31.253-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:40:31.253-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:40:31.254-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:40:31.269-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2025-07-18T12:41:01.773-0500] {processor.py:161} INFO - Started process (PID=4002) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:41:01.773-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:41:01.774-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:41:01.774-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:41:01.777-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:41:01.776-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:41:01.777-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:41:01.777-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:41:01.777-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:41:01.790-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-07-18T12:41:32.226-0500] {processor.py:161} INFO - Started process (PID=4167) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:41:32.227-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:41:32.228-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:41:32.228-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:41:32.230-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:41:32.229-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:41:32.230-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:41:32.230-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:41:32.230-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:41:32.244-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:42:02.754-0500] {processor.py:161} INFO - Started process (PID=4274) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:42:02.755-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:42:02.757-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:42:02.756-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:42:02.760-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:42:02.759-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:42:02.760-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:42:02.760-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:42:02.761-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:42:02.783-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2025-07-18T12:42:33.492-0500] {processor.py:161} INFO - Started process (PID=4383) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:42:33.494-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:42:33.495-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:42:33.495-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:42:33.497-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:42:33.496-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:42:33.497-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:42:33.497-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:42:33.497-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:42:33.512-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-07-18T12:43:03.684-0500] {processor.py:161} INFO - Started process (PID=4492) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:43:03.685-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:43:03.686-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:43:03.686-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:43:03.688-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:43:03.688-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:43:03.688-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:43:03.688-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:43:03.688-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:43:03.701-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2025-07-18T12:43:34.131-0500] {processor.py:161} INFO - Started process (PID=4600) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:43:34.132-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:43:34.133-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:43:34.133-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:43:34.135-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:43:34.135-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:43:34.135-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:43:34.135-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:43:34.135-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:43:34.148-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-07-18T12:44:04.902-0500] {processor.py:161} INFO - Started process (PID=4710) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:44:04.904-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:44:04.905-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:44:04.904-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:44:04.907-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:44:04.906-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:44:04.907-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:44:04.907-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:44:04.907-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:44:04.923-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-07-18T12:44:35.687-0500] {processor.py:161} INFO - Started process (PID=4819) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:44:35.689-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:44:35.690-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:44:35.690-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:44:35.692-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:44:35.691-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:44:35.692-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:44:35.692-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:44:35.692-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:44:35.705-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:45:06.077-0500] {processor.py:161} INFO - Started process (PID=4929) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:45:06.078-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:45:06.079-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:45:06.079-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:45:06.081-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:45:06.081-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:45:06.081-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:45:06.081-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:45:06.082-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:45:06.097-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-07-18T12:45:36.653-0500] {processor.py:161} INFO - Started process (PID=5037) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:45:36.654-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:45:36.655-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:45:36.655-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:45:36.657-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:45:36.657-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:45:36.658-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:45:36.657-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:45:36.658-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:45:36.673-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-07-18T12:46:07.370-0500] {processor.py:161} INFO - Started process (PID=5145) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:46:07.372-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:46:07.373-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:46:07.372-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:46:07.375-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:46:07.374-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:46:07.375-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:46:07.375-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:46:07.375-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:46:07.387-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-07-18T12:46:37.990-0500] {processor.py:161} INFO - Started process (PID=5253) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:46:37.991-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:46:37.992-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:46:37.992-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:46:37.995-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:46:37.994-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:46:37.995-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:46:37.995-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:46:37.995-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:46:38.009-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-07-18T12:47:08.851-0500] {processor.py:161} INFO - Started process (PID=5363) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:47:08.852-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:47:08.854-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:47:08.853-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:47:08.856-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:47:08.855-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:47:08.856-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:47:08.856-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:47:08.856-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:47:08.869-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:47:39.507-0500] {processor.py:161} INFO - Started process (PID=5470) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:47:39.518-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:47:39.519-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:47:39.518-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:47:39.521-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:47:39.520-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:47:39.521-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:47:39.521-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:47:39.521-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:47:39.576-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2025-07-18T12:48:10.136-0500] {processor.py:161} INFO - Started process (PID=5641) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:48:10.138-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:48:10.139-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:48:10.138-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:48:10.140-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:48:10.140-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:48:10.140-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:48:10.140-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:48:10.141-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:48:10.154-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-07-18T12:48:40.810-0500] {processor.py:161} INFO - Started process (PID=5759) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:48:40.810-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:48:40.811-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:48:40.811-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:48:40.813-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:48:40.813-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:48:40.813-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:48:40.813-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:48:40.814-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:48:40.834-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2025-07-18T12:49:11.331-0500] {processor.py:161} INFO - Started process (PID=5868) to work on /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:49:11.332-0500] {processor.py:830} INFO - Processing file /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-07-18T12:49:11.334-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:49:11.333-0500] {dagbag.py:545} INFO - Filling up the DagBag from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:49:11.335-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:49:11.335-0500] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-07-18T12:49:11.336-0500] {logging_mixin.py:188} INFO - [2025-07-18T12:49:11.336-0500] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-07-18T12:49:11.336-0500] {processor.py:842} WARNING - No viable dags retrieved from /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-07-18T12:49:11.350-0500] {processor.py:183} INFO - Processing /home/abhiram/jobtrak-stack/venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
